infrastructure/persistence/src/main/resources/db/migration/V13__create_meeting_member.sql

```diff
@@ -0,0 +1,14 @@
+drop table weave.meeting_team_member;
+create table weave.meeting_member
+(
+    id              binary(16)   not null,
+    meeting_team_id binary(16)   not null,
+    user_id         binary(16)   not null,
+    role            varchar(255) not null,
+    primary key (id),
+    index meeting_member_meeting_team_id_index (meeting_team_id),
```

Comment:
p3) 팀에 유저 추가해서 유니크 인덱스로 걸어두는 것이 더 좋을 것 같아요
```suggestion
    unique index meeting_member_meeting_team_id_and_user_id_uindex (meeting_team_id, user_id),
```